src/main/java/com/arcbees/bourseje/BourseJEDebug.gwt.xml

```diff
@@ -1,9 +1,12 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <module rename-to='bourseje'>
     <inherits name='com.arcbees.bourseje.BourseJE'/>
+    <inherits name="com.google.gwt.logging.Logging"/>
+
+    <add-linker name="xsiframe"/>
```

Comment:
afaik, you get the warning because of https://github.com/ArcBees/gwtquery/blob/master/gwtquery-core/src/main/java/com/google/gwt/query/Query.gwt.xml#L173... So both of you are right, you don't need the linker because it is the default in GWT 2.7. GQuery specifies one, so you actually need to specify it yourself to overwrite GQuery's setting